assets/licensing.txt

```diff
@@ -0,0 +1,2 @@
+Dough.png sourced from https://totalminer.fandom.com/wiki/Dough.  Only change made was resizing
+Licensing is CC BY-SA https://creativecommons.org/licenses/by-nc/3.0/
```

Comment:
Licensing is handled in the implementation document. Please remove and request @sanjsrini or @Boomehx to add it to the new document.